app/javascript/app/pages/planning/planning.js

```diff
@@ -2,8 +2,21 @@ import { connect } from 'react-redux';
 
 import PlanningComponent from './planning-component';
 
-const mapStateToProps = ({ location }) => ({
-  routes: Object.values(location.routesMap).filter(r => !!r.nav)
+const hasSections = sections => Object.keys(sections).length > 0;
+
+const mapStateToProps = ({ location, sections }) => ({
+  routes: Object.values(location.routesMap).filter(r => !!r.nav),
+  selectedCategory: hasSections(sections)
+    ? sections.find(section => section.slug === 'planning').categories[0].slug
+    : null,
+  categories: hasSections(sections)
```

Comment:
We can abstract the 'hasSections' and 'find' in a variable (hasPlanningSection) to reuse them